<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listLive">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listCrum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelCry">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crystals</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDisto">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Distolate</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCrum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crumble/Goo</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCry">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listDisto">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>171</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labeLive">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Live Resin</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPackage">
    <property name="title">
     <string>Package</string>
    </property>
    <addaction name="actionPackage_Selected"/>
   </widget>
   <widget class="QMenu" name="menuSend_To">
    <property name="title">
     <string>Send To</string>
    </property>
    <addaction name="actionSend_to_Bulk"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPackage"/>
   <addaction name="menuSend_To"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPackage_Selected">
   <property name="text">
    <string>Package Selected</string>
   </property>
  </action>
  <action name="actionSend_to_Bulk">
   <property name="text">
    <string>Send to Bulk</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
